engines/grim/lua_v1_graphics.cpp

```diff
@@ -465,10 +470,16 @@ void Lua_V1::KillPrimitive() {
 		return;
 
 	PrimitiveObject *prim = getprimitive(primObj);
-	delete prim;
+	if (prim)
+		delete prim;
 }
 
 void Lua_V1::DimScreen() {
+	lua_Object level = lua_getparam(1);
```

Comment:
why this is added?

Reply:
No idea. It is part of a large commit https://github.com/tobiaspfaff/grim_mouse/commit/5ef5d6789c14be37dd95eaa5e5536097fc96fffa

Reply:
then, please revert. it's for dimming screen